<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MakeVerificationApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(117, 249, 244);
font-family: Noto Sans SC;
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="first_year_lbl">
                <property name="font">
                 <font>
                  <family>Noto Sans SC</family>
                  <pointsize>20</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Год первого сертификата калибровки прибора</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="first_year_line_edit">
                <property name="font">
                 <font>
                  <family>Noto Sans SC</family>
                  <pointsize>25</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="second_year_lbl">
                <property name="font">
                 <font>
                  <family>Noto Sans SC</family>
                  <pointsize>20</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Год второго сертификата калибровки прибора</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="second_year_line_edit">
                <property name="font">
                 <font>
                  <family>Noto Sans SC</family>
                  <pointsize>25</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="method_lbl">
              <property name="font">
               <font>
                <family>Noto Sans SC</family>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Критерии приемки(требования метода)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="method_top_line_edit">
              <property name="font">
               <font>
                <family>Noto Sans SC</family>
                <pointsize>25</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="method_bottom_line_edit">
              <property name="font">
               <font>
                <family>Noto Sans SC</family>
                <pointsize>25</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="file_first_certify_lbl">
                <property name="font">
                 <font>
                  <family>Noto Sans SC</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Путь к первому сертификату</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="file_first_certify_btn">
                <property name="font">
                 <font>
                  <family>Noto Sans SC</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Выбрать файл</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="first_file_path">
              <property name="font">
               <font>
                <family>Noto Sans SC</family>
                <pointsize>25</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Путь к файлу</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="file_second_certify_lbl">
                <property name="font">
                 <font>
                  <family>Noto Sans SC</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Путь ко второму сертификату</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="file_second_certify_btn">
                <property name="font">
                 <font>
                  <family>Noto Sans SC</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Выбрать файл</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="second_file_path">
              <property name="font">
               <font>
                <family>Noto Sans SC</family>
                <pointsize>25</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Путь к файлу</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="save_btn">
          <property name="font">
           <font>
            <family>Noto Sans SC</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Noto Sans SC</family>
            <pointsize>25</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(246, 97, 81)</string>
          </property>
          <property name="text">
           <string>Powered by DarkyZ</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
